eogrow/pipelines/rasterize.py

```diff
@@ -79,44 +43,79 @@ class Schema(Pipeline.Schema):
             description="The storage manager key pointing to the input folder for the rasterization pipeline."
         )
         output_folder_key: str = Field(
-            description="The storage manager key pointing to the output folder for the rasterization pipeline pipeline."
+            description="The storage manager key pointing to the output folder for the rasterization pipeline."
         )
         vector_input: Union[Feature, str] = Field(
             description="A filename in the input_data folder or a feature containing vector data."
         )
         dataset_layer: Optional[str] = Field(
-            description="Name of a layer with data to be rasterized in a multi-layer file."
+            description="In case of multi-layer files, name of the layer with data to be rasterized."
         )
-        columns: List[VectorColumnSchema]
```

Comment:
reducing the layer also brought some parameter name changes, but I believe this change is for the better, since the names are then more intuitive